CHANGELOG.md

```diff
@@ -1,8 +1,13 @@
-# Unreleased
+# 1.0.0
 
+<<<<<<< HEAD
```

Comment:
Looks like the rebase conflict lines are still included

Reply:
Ugh, thanks

Reply:
You'd think I'd save the file I fixed on the rebase, wouldn't you?

Reply:
Should be OK now